mfile/__init__.py

```diff
@@ -1,6 +1,2 @@
-from .erg import ERG
-from .version import __version__
-
-class BSIG:
```

Comment:
I need this dummy class to remain

Reply:
Reverted! (see latest commit)

Sorry about it, I did not get why it was there. Maybe it'd be a good idea to document it's presence on the class docstring.